tests/unit/splinter_test.c

```diff
@@ -129,9 +129,6 @@ CTEST_SETUP(splinter)
    heap_capacity        = MAX(heap_capacity, 2 * GiB);
 
    bool use_shmem = test_using_shmem(Ctest_argc, (char **)Ctest_argv);
-   if (use_shmem) {
-      platform_default_log("Run test using shared memory segment.\n");
```

Comment:
Not needed any more, as CTests' main() also checks and reports when tests are running with `--use-shmem` option.